lib/cloud/aws/ssm_documents.go

```diff
@@ -27,8 +27,12 @@ import (
 // EC2DiscoverySSMDocument receives the proxy address and returns an SSM Document.
 // This document downloads and runs a Teleport installer.
 // Requires the proxy endpoint URL, example: https://tenant.teleport.sh
-func EC2DiscoverySSMDocument(proxy string) string {
-	randString := uuid.NewString() // Secure random so the filename can not be guessed to avoid possible script injection
+func EC2DiscoverySSMDocument(proxy string, insecureSkipNameRandomization bool) string {
+	installTeleportPath := "/tmp/installTeleport.sh"
+	if !insecureSkipNameRandomization {
+		// Secure random so the filename can not be guessed to avoid possible script injection
```

Comment:
Did we previously used to accept random file path input? Looks like this comment was added as part of a fix for that behavior and right now, it does not clearly reflect its meaning. Also secure random to me means random value derived from `crypto/random` package but here we just use uuid so thats a unique random nut not a secure random?

I think we can omit this comment or rephrase it.

Reply:
"secure random" was the comment before

> Did we previously used to accept random file path input

Previously (and currently outside of a test case) we used a uuid appended to the install teleport script path for some security reason.

This change allows a test to skip randomization of the install script path

Reply:
c4392cd3b49e4c02a8034b22c0987e3b1f5f847d